cmd/krel/cmd/release_notes.go

```diff
@@ -48,6 +48,10 @@ const (
 	defaultKubernetesSigsOrg = "kubernetes-sigs"
 	// defaultKubernetesSigsRepo relnotes.k8s.io repository name
 	defaultKubernetesSigsRepo = "release-notes"
+	// defaultKubernetesSigReleaseOrg org owner of the sig-release repo
+	defaultKubernetesSigReleaseOrg = "kubernetes"
+	// defaultKubernetesSigReleaseRepo name of the sig-release repository
+	defaultKubernetesSigReleaseRepo = "sig-release"
```

Comment:
We could avoid that new const when using `git.DefaultGithubReleaseRepo`.

Reply:
Fixed. Also got rid of `defaultKubernetesSigReleaseOrg` using `git.DefaultGithubOrg`

Reply:
Awesome, thank you!